compiler/rustc_codegen_llvm/src/gotoc/operand.rs

```diff
@@ -535,6 +536,7 @@ impl<'tcx> GotocCtx<'tcx> {
                 var.ret(Location::none()),
             ];
             Symbol::function(
+                &fname,
```

Comment:
Is there a better pretty-name to use here?

Reply:
Same as above, removing.